views/random-beer.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="container d-flex">
+    {{>beerpartial}}
+    <div class="d-flex flex-column justify-content-around">
+        <h4 class="card-title">Food Pairings</h4>
+        <ul>
+            {{#each this.food_pairing}}
```

Comment:
juraria que tanto aqui como en la linea 10 no es necesario escribir el this, peor esta perfecto asi